tests/e2e/e2e_test.go

```diff
@@ -1691,3 +1691,48 @@ func (s *IntegrationTestSuite) TestAConcentratedLiquidity_CanonicalPool_And_Para
 	s.Require().NoError(err)
 	osmoassert.DecApproxEq(s.T(), expectedSpotPrice, concentratedPool.GetCurrentSqrtPrice().Power(2), sdk.NewDecWithPrec(1, 3))
 }
+
+func (s *IntegrationTestSuite) TestPoolMigration() {
+	chain := s.configurer.GetChainConfig(0)
+	node, err := chain.GetDefaultNode()
+	s.NoError(err)
+
+	percentOfSharesToMigrate := sdk.MustNewDecFromStr("0.9")
+	fundTokens := []string{"50000000000uosmo", "50000000000stake"}
+	poolJoinAddress := node.CreateWalletAndFund("poolJoinAddress", fundTokens)
+
+	// Get the permisionless pool creation parameter.
+	isPermisionlessCreationEnabledStr := node.QueryParams(cltypes.ModuleName, string(cltypes.KeyIsPermisionlessPoolCreationEnabled))
+	if !strings.EqualFold(isPermisionlessCreationEnabledStr, "false") {
+		s.T().Fatal("concentrated liquidity pool creation is enabled when should not have been")
+	}
```

Comment:
This probably shouldn't be a fail. It should just be a check to see if its enabled, and if its not then submits the param change prop